griptape/drivers/vector/base_vector_driver.py

```diff
@@ -36,6 +36,15 @@ class Entry:
         kw_only=True
     )
 
+    def insert_text_artifacts(self, artifacts: list[tuple[TextArtifact, str, Optional[dict]]], **kwargs) -> None:
```

Comment:
The signature should be the same as `upsert_text_artifacts`. Is there a reason you decided to go for a list of tuples?